packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -225,18 +241,30 @@ export class AbuseService {
           abuseEntity.chainId,
           abuseEntity.escrowAddress,
         );
+        if (!escrow) {
+          throw new Error(
+            `Escrow data is missing. Abuse id: ${abuseEntity.id}`,
+          );
+        }
 
         if (abuseEntity.decision === AbuseDecision.ACCEPTED) {
           await this.slashAccount({
             slasher: abuseEntity?.user?.evmAddress as string,
             staker: escrow.launcher,
-            chainId: abuseEntity.chainId,
-            escrowAddress: abuseEntity.escrowAddress,
+            chainId: chainId,
+            escrowAddress: escrowAddress,
             amount: Number(abuseEntity.amount),
           });
-          const webhookUrl = (
-            await OperatorUtils.getOperator(chainId, escrow.launcher)
-          ).webhookUrl as string;
+          const operator = await OperatorUtils.getOperator(
+            chainId,
+            escrow.launcher,
+          );
+          if (!operator || !operator.webhookUrl) {
+            throw new Error(
+              `Operator data is missing. Abuse id: ${abuseEntity.id}`,
+            );
+          }
```

Comment:
```suggestion
          if (!operator) {
            throw new Error('Operator data is missing');
          }
```
\+ extra check on `webhookUrl` if needed